extensions/kubernetes/openshift/deployment/src/main/resources/config-wrapper.vm

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.kubernetes.deployment;
```

Comment:
What is this file needed for?

Reply:
They are not needed and have been removed.